hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/trigger/Trigger.java

```diff
@@ -25,13 +25,16 @@
 public class Trigger extends TriggerTemplate {
 
     public enum Mode {
-        FIRING, SAFETY
+        FIRE, SAFETY
     };
 
     private String id;
     private boolean enabled;
+    private boolean safetyEnabled;
     private Mode mode;
 
+    private transient Match match;
```

Comment:
I have an open question and I don't have a good answer for that.
I wonder if the public api.model would be more DTO oriented rather than have hidden logic.
I mean, even if we mark some properties as transient, those are there and they are part of the REST interface.
So I wonder if we should have a single model with everything (public properties or private properties used by the engine but not needed from the API) or may be to split model, and to have a plubic model very simple and POJO oriented that will be used for integration, and have private/hidden properties in a private model used by the engine.
Both options have pros/cons, so I don't have an answer but I bring it to the review.